src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/sql_request.yml

```diff
@@ -44,7 +44,7 @@ admin_sql_requests_edit:
 
 admin_sql_requests_delete:
     path: /{sqlRequestId}/delete
-    methods: [GET]
+    methods: [POST]
```

Comment:
I believe this is a BC break, either we allow both GET and POST
Or at least we need to mark this as BC

Reply:
I put back the GET verb and add DELETE